src/api/apicalls.js

```diff
@@ -0,0 +1,167 @@
+import axios from "axios";
+const auth =
+	"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiIxZjY1YzdlMC1lNWQyLTQ1NTktOWI4Mi04OTE4MmE1NTU2YjgiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.C48kdE1B0-zI6Gbtl7EG_FE2bNBezhXGB9yur-OvcwQ";
+const headers = { authorization: auth };
+
+const signupuser = async (user) => {
+	try {
+		const response = await axios.post("/api/auth/signup", { user });
+		return { token: response.data.encodedToken, success: true };
+	} catch (err) {
+		return { token: "", success: false };
+	}
+};
+const loginuser = async (user) => {
+	try {
+		const response = await axios.post("/api/auth/login", { user });
+		return { token: response.data.encodedToken, success: true };
+	} catch (err) {
+		return { token: "", success: false };
+	}
+};
+
+const getAllVideos = async () => {
+	try {
+		const response = await axios.get("/api/videos");
```

Comment:
suggestion : here api call could have been generalised in order to maintain DRY.

Reply:
I will try to generalize api calls once i complete all p0. Thanks for suggestion.